README.md

```diff
@@ -68,7 +85,9 @@ docker build -t firecloud-orch .
 Run the container:
 ```
 docker run --name orch \
+  -p 8080:8080
+  -e GOOGLE_CLIENT_ID='$GOOGLE_CLIENT_ID' \
   -e RAWLS_URL_ROOT='https://rawls.dsde-dev.broadinstitute.org' \
   -e AGORA_URL_ROOT='https://agora.dsde-dev.broadinstitute.org' \
   firecloud-orch
-```
\ No newline at end of file
+```
```

Comment:
Running this once seemed to work (container started, etc.), but killing and re-running failed. I get a container error, even though no container is running. Removing the --name option works.

```
wmd08-a62:firecloud-orchestration grushton$ docker run --name orch   -p 8080:8080   -e GOOGLE_CLIENT_ID='$GOOGLE_CLIENT_ID'   -e RAWLS_URL_ROOT='https://rawls.dsde-dev.broadinstitute.org'   -e AGORA_URL_ROOT='https://agora.dsde-dev.broadinstitute.org'   firecloud-orch
Error response from daemon: Conflict. The name "orch" is already in use by container eed056eeea43. You have to delete (or rename) that container to be able to reuse that name.
wmd08-a62:firecloud-orchestration grushton$ docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
wmd08-a62:firecloud-orchestration grushton$ 
```